pkg/controllers/store_controller.go

```diff
@@ -115,17 +111,13 @@ func storeAddOrReplace(spec configv1beta1.StoreSpec, fullname string) error {
 		return fmt.Errorf("store factory failed to create store from store config, err: %w", err)
 	}
 
-	StoreMap[fullname] = storeReference
+	// TODO: pass the actual namespace once multi-tenancy is supported.
+	StoreMap.AddStore(constants.EmptyNamespace, fullname, storeReference)
 	logrus.Infof("store '%v' added to store map", storeReference.Name())
```

Comment:
Not exactly related to this PR, but do you feel we should revisit some the level of the log statements in the controllers? I'm wondering how useful is it to have more than one statement as Info per reconcile operation. For example, here, when a user updates/deletes the store resource, the store controller will first output "reconciling store" and then also "store add to store map".

Reply:
Good question, I feel for controller logs, it's ok to have more info logs as reconcile does not often happen and could make debug easier. But we do need to revisit what information should be logged. Like the message `reconciling store` does not provide much information to users.